wallet/lib/services/third_party_services/remote_config_service/remote_config_service.dart

```diff
@@ -18,6 +18,9 @@ abstract class RemoteConfigService {
 
   /// This method returns the ios app version in the remote config
   String getIOSAppVersion();
+
+  /// This method returns whether the app is in maintenance mode or not
```

Comment:
Update documentation it should be like this
/// This method tells whether the app is in maintenance mode or not
/// Output: return [bool] true if in maintenance mode else will return false